config/model_configs/box_density_current_test.yml

```diff
@@ -1,16 +1,21 @@
-dt_save_state_to_disk: "10secs"
 initial_condition: "DryDensityCurrentProfile"
+config: "box"
+smagorinsky_lilly: true
+c_smag: 0.25
+discrete_hydrostatic_balance: true
+hyperdiff: "false"
 x_max: 51200.0
+y_max: 51200.0
+z_max: 6400.0
+x_elem: 45
+y_elem: 45
```

Comment:
Resolution is rather coarse but this is fine as an integration test for regular CI I think.